### PR TITLE
[native] Remove unused PrestoExchangeSource::supportsFlowControl API

### PR DESCRIPTION
**Pull request by mbasmanova**
```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Review by gggrace14 — APPROVED** [review]

**Review by amitkdutta — APPROVED** [review]
Thanks @mbasmanova

**Comment by aditi-pandit** [comment]
@mbasmanova : Just confirming the two changes in this PR are unrelated right ?

**Comment by mbasmanova** [comment]
> @mbasmanova : Just confirming the two changes in this PR are unrelated right ?

@aditi-pandit Aditi, which two changes are you referring to? There are 2 commits. One is to disable flaky test. I added it because the CI failed.

**Review by pranjalssh — APPROVED** [review]

File: presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java [hunk 1]
```diff
@@ -390,6 +390,7 @@ protected void assertCreateTableAsSelect(Session session, String table, @Languag
         assertFalse(getQueryRunner().tableExists(session, table));
     }
 
+    // Flaky test: https://github.com/prestodb/presto/issues/20764
```
Comment (majetideepak) [inline presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java:393]: Should we add `enable = false` inside `@Test` ?

**Review by majetideepak — APPROVED** [review]

**Pull request merged**